DetectorDescription/Detectors/compact/FCChh_ECalBarrel_Mockup.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
+
+  <includes>
+    <gdmlFile  ref="elements.xml"/>
+    <gdmlFile  ref="materials.xml"/>
+  </includes>
+
+  <info name="FCChh_ECalBarrel_Mockup"
+    title="ECal Barrel Calorimeter"
+    author="A.Dell'Acqua"
+    url="no"
+    status="development"
+    version="$Id:??$">
+    <comment>ECal very conceptual design</comment>
+  </info>
+
+  <define>
+    <!-- World volume --->
+    <constant name="world_size" value="30*m"/>
+    <constant name="world_x" value="world_size"/>
+    <constant name="world_y" value="world_size"/>
+    <constant name="world_z" value="world_size"/>
+
```

Comment:
Why is the world volume defined in the detector xml?
I think it is better to define it once in a "master" xml as already done (FCChh_DectMaster.xml)